app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.kt

```diff
@@ -0,0 +1,43 @@
+package org.hackillinois.android.view.schedule
+
+import android.os.Bundle
+import android.support.design.widget.TabLayout
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentPagerAdapter
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.fragment_admin.view.*
+import org.hackillinois.android.R
+import org.hackillinois.android.viewmodel.ScheduleViewModel
+
+class ScheduleFragment : Fragment() {
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view = inflater.inflate(R.layout.fragment_schedule, container, false)
+
+        val sectionsPagerAdapter = SectionsPagerAdapter(childFragmentManager)
+
+        view.adminContainer.adapter = sectionsPagerAdapter
+        view.adminContainer.addOnPageChangeListener(TabLayout.TabLayoutOnPageChangeListener(view.tabs))
+        view.tabs.addOnTabSelectedListener(TabLayout.ViewPagerOnTabSelectedListener(view.adminContainer))
+
+        val scheduleViewModel = ScheduleViewModel()
+
+        val time = System.currentTimeMillis()
+
+        view.adminContainer.currentItem = when {
+            time < scheduleViewModel.fridayEnd -> 0
+            time < scheduleViewModel.saturdayEnd -> 1
+            else -> 2
```

Comment:
So after the event ends, the schedule is still stuck on Sunday... it should probably go back to Friday